spec/controllers/annotation_lists_controller_spec.rb

```diff
@@ -3,223 +3,61 @@
 RSpec.describe AnnotationListsController, type: :controller do
 
   before(:all) do
-    @annoLayer ='{"layer_id": "http://localhost:5000/layers/testLayer", "label": "Layer 2", "motivation": "yale:transcribing", "license": "http://creativecommons.org/licenses/by/4.0/", "layer_type": "sc:Layer", "@context": "http://iiif.io/api/presentation/2/context.json"}'
-    @annotationLayer = AnnotationLayer.create(JSON.parse(@annoLayer))
-  end
-
-  context 'when Post is called' do
-    describe 'POST annotation json' do
-        before(:each) do
-          @annoListString ='{"@type": "sc:AnnotationList","@context": "http://iiif.io/api/presentation/2/context.json", "label":"transcription layer 2 list 2","within":["http://localhost:5000/layers/testLayer"]}'
-        end
-
-      it 'returns a 201 ("created") response' do
-        post :create, JSON.parse(@annoListString), :format => 'json', :Content_Type => 'application/json', :Accept => 'application/json'
-        expect(response.status).to eq(201)
-      end
-
-      it 'creates a new List' do
-        expect { post :create, JSON.parse(@annoListString) }.to change(AnnotationList, :count).by(1)
-      end
-
-      it 'creates an @id for the returned list' do
-        post :create, JSON.parse(@annoListString)
-        expect(response.status).to eq(201)
-        json = JSON.parse(response.body)
-        expect(json['@id']).to be_truthy
-      end
-
-      it 'assigns the version' do
-        post :create, JSON.parse(@annoListString)
-        @annotationList = AnnotationList.last()
-        expect(@annotationList['version']).to eq (1)
-      end
-
-      it 'fails validation if motivation is nil' do
-        json = JSON.parse(@annoListString)
-        json['label'] = nil
-        post :create, json
-        expect(response.status).to eq(422)
-      end
-
-      it 'does not fail validation if within is nil' do
-        json = JSON.parse(@annoListString)
-        json['within'] = nil
-        post :create, json
-        expect(response.status).to eq(201)
-      end
-
-      it 'updates the map if within is not nil' do
-        json = JSON.parse(@annoListString)
-        post :create, json
-        json = JSON.parse(response.body)
-        layers = LayerListsMap.getLayersForList json['@id']
-        expect(layers).not_to eq(nil)
-        expect(layers).to eq(json['within'])
-      end
-
-    end
+    layer = {
+      '@id' => 'http://localhost:5000/layers/testLayer',
+      '@type' => 'sc:Layer',
+      '@context' => 'http://iiif.io/api/presentation/2/context.json',
+      'label' => 'Layer 2'
+    }
+    @annotationLayer = AnnotationLayer.create_from_iiif(layer)
   end
 
   context 'when Get is called' do
     describe 'GET annotation json' do
       before(:each) do
         AnnotationList.delete_all
-        layer_obj = IIIF::Layer.create(id: '/layer/1')
-        AnnotationLayer.create_from_iiif(layer_obj)
+        AnnotationLayer.delete_all
```

Comment:
I believe this is what the DatabaseCleaner gem takes care of right? https://github.com/yale-web-technologies/mirador-annotations/blob/5f3b5c525c9daad3be040d329584b5b49a8b05d5/spec/rails_helper.rb#L68